ui/shared/polyfill.number.toLocaleString.js

```diff
@@ -273,22 +273,20 @@ function toLocaleString(val, locale, options) {
 
     sNum = mapMatch(transformForLocale, locale)(sNum, options); 
 
-    if(options && options.currency && options.style === "currency") { 
-        var format = currencyFormats[mapMatch(currencyFormatMap, locale)]; 
-        var symbol = currencySymbols[options.currency.toLowerCase()]; 
-        if(options.currencyDisplay === "code" || !symbol) { 
-            sNum = renderFormat(format, { 
```

Comment:
Is renderFormat dead code now?

Reply:
I did not see anyone use it on those formats, kept it in case a requirement like that comes back? @alaibe

Reply:
IMO it's better to delete code that isn't use. We still have git to recover things if needed

Reply:
its is ok to me either ways. will remove it and push it again!

Reply:
Hey @alaibe Done :)